velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -822,14 +1063,32 @@ TEST_F(DecodedVectorTest, dictionaryWrapping) {
     // Get wrap and nulls from the decoded vector and ensure they are correct,
     // wrap base vector with them and compare initial dictionary and the new
     // one.
-    decoded.decode(*dict, rows);
-    auto wrapping = decoded.dictionaryWrapping(*dict, rows);
-    auto wrapped = BaseVector::wrapInDictionary(
-        std::move(wrapping.nulls),
-        std::move(wrapping.indices),
-        rows.end(),
-        baseWithNulls);
-    assertEqualVectors(dict, wrapped);
+    {
+      SelectivityVector rows(outerDictSize);
+      DecodedVector decoded;
+
+      decoded.decode(*dict, rows);
+      auto wrapping = decoded.dictionaryWrapping(*dict, rows);
```

Comment:
any particular reason not to use 'check' lambda?

Reply:
here the check is only one line       assertEqualVectors(dict, wrapped);